model/labels/labels.go

```diff
@@ -485,3 +486,8 @@ func (b *ScratchBuilder) Labels() Labels {
 func (b *ScratchBuilder) Overwrite(ls *Labels) {
 	*ls = append((*ls)[:0], b.add...)
 }
+
+// SizeOfLabels returns the approximate space required for n copies of a label.
+func SizeOfLabels(value string, n uint64) uint64 {
```

Comment:
Given that the size is stored in `labelValueLength/LabelValueStats` I thought we should only consider the values.
But because the values cannot exist without the names and every implementation stores the names differently, maybe we should account for the names as well, in that case we may need to adjust the naming.